src/components/ObsDetails/ObsDetails.js

```diff
@@ -202,8 +203,16 @@ const ObsDetails = ( ): Node => {
       />
     );
 
+    const backButton = ( ) => (
+      <HeaderBackButton
+        tintColor={colors.black}
+        onPress={navigation.goBack}
+      />
+    )
+
     navigation.setOptions( {
-      headerRight: editIcon
+      headerRight: editIcon,
+      headerLeft: backButton
```

Comment:
I'm not sure how this page had a back button w/o this code previously. But for what it's worth, I think that this should be defined w/ the screen definition in the navigator.

Reply:
Can do, for future reference, in a Stack navigator, the back button gets automatically added if it is possible to go back. But when we switched to tab navigation, the behavior did not follow